src/main/kotlin/lotto/domain/LottoGenerator.kt

```diff
@@ -12,9 +12,10 @@ class LottoGenerator {
         return lottoNumbers.map { LottoNumber(it) }
     }
 
-    fun generateLottos(count: Int): Lottos {
+    fun generateLottos(count: Int, manualLottos: List<LottoNumbers>): Lottos {
```

Comment:
```suggestion
    fun generateLottos(count: Int, manualLottos: Lottos): Lottos {
```

로 변경해볼까요!?

추가로 이제 LottoNumbers 클래스를 Lotto로 변경해도 괜찮지 않을까합니다~

LottoNumber가 모여서 Lotto가 되는 것처럼 말이죠 😊

Reply:
그렇네요 ㅎㅎ
변경해보겠습니다~